lte/protos/session_manager.proto

```diff
@@ -328,8 +334,16 @@ message ChargingCredit {
   FinalAction final_action = 5;
   GrantedUnits granted_units = 6;
   RedirectServer redirect_server = 7;
+  repeated string restrict_rules = 8;
+}
+
+message RequestedUnits {
+  uint64 total = 1;
+  uint64 tx = 2;
+  uint64 rx = 3;
```

Comment:
are those values going to be always sent. Or can we have a situation were only `tx` is sent, or `tx` and `rx` are sent?  

I am asking this because protos doesn't have null values. So anything that is not sent will be defaulted to 0. 

For the communication between PCRF and sessiond we implemented this:

```
message CreditUnit {
  bool is_valid = 1;
  uint64 volume = 2;
}

message GrantedUnits {
  CreditUnit total = 1;
  CreditUnit tx = 2;
  CreditUnit rx = 3;
}
```

Reply:
This is the proto definition syntax. Those values are already present in session_manager proto of LTE.

Reply:
@uri200 , I can resolve this comment.

Reply:
Understood. Then there is no change on 5g. To answer the question: yes all TOTAL, TX and RX will be sent to the core